azext_edge/edge/providers/adr/schemas/opcua_connector_metadata.json

```diff
@@ -0,0 +1,429 @@
+{
+    "$schema": "https://raw.githubusercontent.com/Azure/iot-operations-sdks/refs/heads/main/doc/akri_connector/connector-metadata-schema.json",
+    "name": "AzureIoTOperationsOPCUAConnector",
+    "description": "Azure IoT Operations connector for OPC UA.",
+    "version": "1.2.82",
+    "isPreview": false,
+    "maintainer": "mschierd@microsoft.com",
+    "vendor": "Microsoft",
+    "imageConfigurationSettings": {
+        "imageName": "aio-connectors/supervisor",
+        "tag": "1.2.82"
+    },
+    "supportedArchitectures": [
+        "linux/amd64"
+    ],
+    "endpointsEnabledByDefault": true,
+    "inboundEndpoints": [
+        {
+            "endpointType": "Microsoft.OpcUa",
+            "supportedAuthenticationTypes": [
+                "usernamePassword",
+                "anonymous"
+            ],
+            "description": "An OPC UA endpoint",
+            "assetsEnabledByDefault": true,
+            "fields": {
+                "address": {
+                    "input": "required",
+                    "exampleValue": "opc.tcp://your-opcua-server-address:4840/",
+                    "description": "The address of the OPC UA server to connect with in the format: <tcp address>:<port>/<path>."
+                }
+            },
+            "additionalConfigurationSchema": {
+                "$schema": "http://json-schema.org/draft-07/schema#",
+                "$id": "https://azure-iot-operations/schemas/assetendpointprofile/additionalconfiguration/opcua/1.3.0.json",
+                "title": "AIO OPC UA Device Inbound Endpoint Additional Configuration Schema",
+                "description": "Schema for the additional configuration of OPC UA inbound endpoint of an device in Azure Industrial Operations",
+                "type": "object",
+                "properties": {
+                    "applicationName": {
+                        "type": "string",
+                        "default": "OPC UA Broker",
+                        "description": "Name that is used if AIO OPC UA Connector needs to create own (self-signed) certificate"
+                    },
+                    "keepAliveMilliseconds": {
```

Comment:
There are json schemas for asset types here https://github.com/Azure/azure-iot-ops-cli-extension/blob/dev/azext_edge/edge/providers/adr/specs.py , they are python dicts over json files which has some benefit.